### ISSUE TITLE
Can I use this package to record audio and play it with slider?

### ISSUE THREAD

**Issue by muhammadwafa** [react-native-community/react-native-audio-toolkit#231]
I have a chat application, and there in chat box people records their audio and send to their friends, so their friends will play the audio by clicking on play button then the slider is moving forward.

Question is, is that possible with this package ???

Already I was using `react-native-audio` and `react-native-sound`, but it is more than two years they never updated these packages and also they mentioned we are not maintaining any more. because that I am switching to some updated packages.

**Comment by muhammadwafa** [comment]
any help please...

**Comment by disrae** [comment]
yes, I do have the slider working, but struggling to get record to work on ios simulator....

**Comment by Muhsin-Abbas** [comment]
in which language used for chat app???
